index.js

```diff
@@ -289,4 +285,24 @@ async function handlePun(message) {
 		message.channel.send('The joke is that this API doesn\'t work.');
 		console.log(error);
 	});
+
+	async function YOLOStatus(CompanyName) {
```

Comment:
JavaScript function names and variables always start with a lowercase letter. 
YOLOStatus -> yoloStatus 
CompanyName -> companyName
etc.